packages/analytics/analytics-geo-map/src/components/AnalyticsGeoMap.vue

```diff
@@ -79,6 +79,7 @@ const {
   showTooltipValue = true,
   fitToCountry = undefined,
   bounds = undefined,
+  truncatedMetric = true,
```

Comment:
Hmm, I'm not sure we want to do this through another prop. 

We'd have to update the dashboard schema as well and then everywhere a dashboard or geo map is used, we'd need logic for figuring out when to truncate or not. 

Is there some default behaviour we can implement internally to decide if the metric should be truncated, rather than introducing a new prop? I think analytics-chart does something like that